### ISSUE TITLE
How to create multiple adaptationset from single file?

### ISSUE THREAD

**Issue by bakamaru** [axiomatic-systems/Bento4#332]
Hi,I'm new to dash technology. I tried to convert video to dash file and its working but my problem is 
i want to create multiple AdaptationSet in mpd file (eg. 480,720,1080) with original single input video file.
How can i achieve that?
my current mpd files contents 
<?xml version="1.0" ?>
<MPD mediaPresentationDuration="PT1M51.407S" minBufferTime="PT35.00S" profiles="'asdf,asdfe3,asdf'" type="static" xmlns="urn:mpeg:dash:schema:mpd:2011">
  
  <Period>
    
    <AdaptationSet maxHeight="1080" maxWidth="1920" mimeType="video/mp4" minHeight="1080" minWidth="1920" segmentAlignment="true" startWithSAP="1">
      <SegmentTemplate duration="35000" initialization="$RepresentationID$/init.mp4" media="$RepresentationID$/seg-$Number$.m4s" startNumber="1" timescale="1000"/>
      <Representation bandwidth="17183449" codecs="avc1.640029" frameRate="500850/16711" height="1080" id="video/avc1" scanType="progressive" width="1920"/>
    </AdaptationSet>
    
    <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1">
      <SegmentTemplate duration="35000" initialization="$RepresentationID$/init.mp4" media="$RepresentationID$/seg-$Number$.m4s" startNumber="1" timescale="1000"/>
      <Representation audioSamplingRate="44100" bandwidth="64607" codecs="mp4a.40.2" id="audio/und/mp4a">
        <AudioChannelConfiguration schemeIdUri="urn:mpeg:dash:23003:3:audio_channel_configuration:2011" value="1"/>
      </Representation>
    </AdaptationSet>
  </Period>
</MPD>

**Comment by barbibulle** [comment]
Do you have a single mp4 file with multiple video tracks, each with a different resolution?

**Comment by bakamaru** [comment]
Yes,it had but i find very complex it self so i tried single multiple video files and again found Video tracks are not aligned error and in issues https://github.com/axiomatic-systems/Bento4/issues/263 you've said it has been fixed but not pushed to master. 
So when it will be pushed?
